src/components/va-accordion/va-accordion.e2e.ts

```diff
@@ -29,10 +29,10 @@ describe('va-accordion', () => {
     await axeCheck(page);
   });
 
-  it('closes one item when another opens', async () => {
+  it('closes one item when another opens if `multi` is false', async () => {
     const page = await newE2EPage();
     await page.setContent(`
-      <va-accordion>
+      <va-accordion multi="false">
```

Comment:
I typically prefer truthy props. So rather than `multi="false"`, I'd recommend `open-single` or something.

Reply:
That seems reasonable - I'll make the change.